src/converters/scalars.rs

```diff
@@ -0,0 +1,155 @@
+use crate::converters::traits::Converter;
+use anyhow::{Error, Result};
+use rerun;
+use std::io::Cursor;
+use std::sync::Arc;
+
+// Converter for std_msgs/msg/Int8.msg
+pub struct Int8Converter {}
+
+impl Converter for Int8Converter {
+    fn convert(
+        &self,
+        rec: &Arc<rerun::RecordingStream>,
+        cdr_buffer: &mut Cursor<Vec<u8>>,
+    ) -> Result<(), Error> {
+        let value = cdr::deserialize_from::<_, i8, _>(cdr_buffer, cdr::Infinite)?;
+        rec.log("scalar", &rerun::Scalar::new(value as f64))?;
```

Comment:
All of these should use the entity_path from the `Conversion` as the log destination.